dockerfiles/verify-sentry/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM sentry:9.0
+RUN pip install https://github.com/getsentry/sentry-auth-github/archive/master.zip
```

Comment:
is it worth considering pinning a specific version rather than `master`?

Reply:
the last commit was `Sep 22, 2017` though... might be worth just in case...

Reply:
Also, would a `requirements.txt` file be applicable here? I think GitHub should scan it to check for vulnerable dependencies.

Reply:
👍 that would also have the benefit of pinning all transitive dependencies, not just the sentry-auth-github one

Reply:
sentry-auth-github doesn't have any dependencies... but yeah, we can use the requirements.txt file. And then we can switch to the -onbuild image...

Reply:
Resolved this to use the requirements.txt and the onbuild version of Sentry.  Pegged the version of the Sentry-Auth-Github to the current commit version on Github.